wiki/osu!api/fr.md

```diff
@@ -1,15 +1,15 @@
----
-outdated: true
-outdated_since: f90e775380f1c7041a62f3d7f5e3e29ba4326dee
----
-
 # osu!api
 
-osu! possède une API publique que les services tiers peuvent utiliser pour développer et prendre en charge la plateforme osu!. Auparavant, il était plus privé et ne pouvait être utilisé que sur demande, mais tout le monde peut désormais utiliser l'API tant qu'il possède un compte osu!.
+osu! possède une [API](https://fr.wikipedia.org/wiki/API) publique que des services tiers peuvent utiliser pour développer et soutenir la plateforme osu!. C'était plus privé, on ne pouvais l'utiliser que sur demande, mais maintenant tout le monde peut utiliser l'API tant qu'il a un compte osu!.
+
+Vous pouvez trouver la documentation de l'API sur [Wiki d'osu-api](https://github.com/ppy/osu-api/wiki). La version actuelle de l'API utilise des clés privées pour authentifier les demandes. Vous pouvez demander une clé API sur la [page d'application](https://osu.ppy.sh/p/api) sur le site d'osu!. Notez que la documentation sur le wiki osu-api n'est peut-être pas entièrement complète ni étendu et que l'API elle-même n'est pas en cours de développement actif. Une nouvelle API, beaucoup plus robuste, est en préparation sur [osu-web](https://github.com/ppy/osu-web).
```

Comment:
last sentence was removed in English, because this is explained in the next paragraph